### PR TITLE
Fix for Css files in .vue files

### LINKED ISSUE #6546: Can't add css to .vue
# 🐛 bug report

I tried adding some css to my apps .vue file, but I can't find a way with which it works.

## 🎛 Configuration (.babelrc, package.json, cli command)

package.json: 
```json
{
	"name": "...",
	"version": "...",
	"description": "",
	"dependencies": {},
	"repository": {
		"type": "git",
		"url": "..."
	},
	"author": "",
	"license": "ISC",
	"bugs": {
		"url": "..."
	},
	"homepage": "...",
	"devDependencies": {}
}

```

## 🤔 Expected Behavior

Parcel adds the files.

## 😯 Current Behavior

I've tried different things (all with different errors), but it currently outputs
```
× Build failed.
@parcel/transformer-vue: Unknown style language: ".css"
```
Pretty sure it should know css, but I also tried .scss and that one failed too.

## 💻 Code Sample

index.htm: 
```html
<!doctype html>
<html>
	<head>
		<title>...</title>
	</head>
	<body>
		<div id="app"></div>
		<script src="./index.js"></script>
	</body>
</html>
```
index.js: 
```javascript
import { createApp } from 'vue';
import App from './App.vue';

const vapp = createApp(App);
vapp.mount('#app');
```
App.vue: 
```vue
<template>
	<div id="content">
		<p>testing</p>
	</div>
</template>
<style src="./App.css"></style>
<script lang="ts">
	export default {
		data() {
			return {
			};
		}
	};
</script>
```
App.css (example): 
```css
#content {
	color: red;
}
```

Inline css does not work either, but adding the css in the html file instead of the vue file works.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.0.0-beta.3.1 |
| Vue              | 3.1.4 |
| @parcel/transformer-vue | 2.0.0-beta.3.1 |
| Node             | 6.14.13 |
| Operating System | Windows 11 (Insider Build) |

### PR DESCRIPTION
**Pull request by sda3-14**
# ↪️ Pull Request

Css files imported in .vue files didn't work, since it tried to check for e.g. "css", when it should check for ".css".

Closes https://github.com/parcel-bundler/parcel/issues/6546

## ✔️ PR Todo

- [ ] Added/updated unit tests for this change
- [ ] Filled out test instructions (In case there aren't any unit tests)
- [ ] Included links to related issues/PRs

### CODE REVIEW

File: packages/transformers/vue/src/VueTransformer.js [hunk 1]
```diff
@@ -320,12 +320,12 @@ ${
             style.lang = extname(style.src);
           }
           switch (style.lang) {
-            case 'less':
-            case 'stylus':
-            case 'styl':
-            case 'scss':
-            case 'sass':
-            case 'css':
+            case '.less':
```
Comment (DeMoorJasper) [inline packages/transformers/vue/src/VueTransformer.js]: can you instead trim of the `.` so it still matches without the dot

**Review by DeMoorJasper — CHANGES_REQUESTED** [review]

File: packages/transformers/vue/src/VueTransformer.js (hunk 1 above)
Comment (sda3-14) [reply packages/transformers/vue/src/VueTransformer.js]: Ok

**Review by DeMoorJasper — APPROVED** [review]

**Comment by mischnic** [comment]
It would be great to have a test case for referencing an external styles file from a vue file, similar to this one https://github.com/parcel-bundler/parcel/blob/81293fc84517f756b593a73d94da4e2630339140/packages/core/integration-tests/test/vue.js#L22-L36

**Comment by 101arrowz** [comment]
Can we supersede with #7069?

**Pull request closed without merging**